src/components/modals/UnwrapModal/index.tsx

```diff
@@ -53,6 +53,28 @@ export const UnwrapModal: React.FC<Props> = (props) => {
     }
   }, [maxBalance])
 
+  const isSubmitDisabled = amount.isZero()
+
+  const unWrap = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.KeyboardEvent<HTMLInputElement>
+  ) => {
+    const unWrapValues = {
+      amount,
+      address: CTService.address,
+      positionId,
+    }
+
+    if (isSubmitDisabled) return
+
+    onUnWrap(unWrapValues)
+
+    if (onRequestClose) onRequestClose(e)
+  }
+
+  const onPressEnter = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') unWrap(e)
+  }
```

Comment:
Render a button is cheap, but anyways...you can wrap `unwrap` in a `useCallback` hook, but it's ok without doing it

Reply:
Adding `useCallback` was even cheaper. Fixed.